extra/1-currency-conversion.js

```diff
@@ -15,7 +18,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let priceToBeConverted = 0.99*price;
+  let convertedPrice = priceToBeConverted * 5.7;
+  return parseFloat(convertedPrice.toFixed(2));
```

Comment:
Nice job with the parseFloat(). I just got to know about it from reviewing your codes... Great job Farzad!